src/components/common/SafeTokenWidget/index.tsx

```diff
@@ -50,14 +63,37 @@ const SafeTokenWidget = () => {
             <Link href={url || ''} passHref legacyBehavior>
               <ButtonBase
                 aria-describedby="safe-token-widget"
-                sx={{ alignSelf: 'stretch' }}
+                sx={canRedeemSep5 ? { height: '42px !important' } : undefined}
                 className={css.tokenButton}
```

Comment:
nit: You can also add this via css and avoid the !important e.g. `className={classnames(css.tokenButton, { [css.sep5]: canRedeemSep5 })`